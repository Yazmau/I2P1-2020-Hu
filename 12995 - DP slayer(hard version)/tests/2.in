10 100000000
25000000 99999506
25000000 99999432
1 0
7 10
25000000 99999511
25000001 100000000
25000001 100000000
25000001 100000000
25000000 99999866
8 2
100
1 2
1 3
1 4
1 5
1 6
1 7
1 8
1 9
1 10
2 3
2 4
2 5
2 6
2 7
2 8
2 9
2 10
3 4
3 5
3 6
3 7
3 8
3 9
3 10
4 5
4 6
4 7
4 8
4 9
4 10
5 6
5 7
5 8
5 9
5 10
6 7
6 8
6 9
6 10
7 8
7 9
7 10
8 9
8 10
9 10
5 7
8 10
4 7
3 5
1 9
5 9
5 7
4 8
3 5
4 9
3 7
5 6
5 10
7 10
8 10
9 10
2 4
9 10
4 9
6 9
2 10
3 6
6 7
1 5
9 10
2 6
7 10
2 8
4 9
3 8
2 6
3 4
9 10
3 4
8 9
2 6
8 9
9 10
2 9
2 9
4 5
8 9
6 8
8 9
9 10
5 7
2 9
9 10
3 10
2 7
8 9
8 10
5 10
7 10
7 9
