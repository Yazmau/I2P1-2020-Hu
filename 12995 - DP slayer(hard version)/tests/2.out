399998315
299999518
299999518
399998315
299999518
299999518
299999518
399998315
299999518
299999444
299999444
399998315
299999444
299999444
299999444
399998315
299999444
300000012
299999523
300000012
300000012
300000012
299999878
300000012
299999523
300000012
300000012
300000012
299999878
300000012
299999523
299999523
299999523
399998315
299999523
300000012
300000012
299999878
300000012
300000012
299999878
300000012
299999878
300000012
299999878
299999523
300000012
300000012
299999523
399998315
399998315
299999523
300000012
299999523
299999878
300000012
299999523
299999523
300000012
300000012
299999878
299999444
299999878
299999878
299999878
299999444
300000012
300000012
399998315
299999878
299999444
300000012
299999444
299999878
300000012
299999444
300000012
299999878
300000012
299999878
299999444
299999878
299999878
399998315
399998315
299999523
299999878
300000012
299999878
299999878
299999523
399998315
299999878
300000012
299999444
299999878
300000012
299999523
300000012
299999878
